changelogs/fragments/add-sriov-advanced-options-to-vmware_guest_network.yml

```diff
@@ -0,0 +1,4 @@
+bugfixes:
+  - _get_nics_from_vm - acquire physical/virtual backing info and guest OS MTU status for diff generation
+  - _new_nic_spec - add advanced SR-IOV options from the VMware API (PCI dev PF/VF backing and guest OS MTU change)
+  - main - add support for the extra arguments
```

Comment:
Here, too, it might be better to use `minor_changes`. It's adding new functionality.

On the other hand, the first two entries look like something you've done in order to implement adding advanced SR-IOV options and this is quite low-level. Our changelogs are more focused on the users of this collection. Maybe it would be best to have just one changelog fragment with:

```
minor_changes:
  vmware_guest_network - Add parameters `physical_function_backing`, `virtual_function_backing` and `allow_guest_os_mtu_change`(https://github.com/ansible-collections/community.vmware/pull/1218)
```

Reply:
Understood, I removed my other fragment file from PR #1080 and fixed my new fragment as per the above.